### PR TITLE
New enrollment options

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c [hunk 1]
```diff
@@ -1428,23 +1435,27 @@ static char* config_file;
...
 static int parse_enroll_opts(int argc, char *argv[]) {
     static struct option opts[] = {
-            { "jwt", required_argument, NULL, 'j'},
-            { "identity", required_argument, NULL, 'i'},
-            { "use-keychain", no_argument, NULL, 'K' },
-            { "key", required_argument, NULL, 'k'},
-            { "cert", required_argument, NULL, 'c'},
-            { "name", required_argument, NULL, 'n'},
-            { "proxy", required_argument, NULL, 'x' },
+        {"url", required_argument, NULL, 'u'},
```
Comment (dovholuknf) [inline programs/ziti-edge-tunnel/ziti-edge-tunnel.c:1438]: the lack of a space on this one triggered me :)

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c [hunk 2]
```diff
@@ -2312,16 +2365,18 @@ static int add_identity_opts(int argc, char *argv[]) {
...
-        "\t-x|--proxy type://[username[:password]@]hostname_or_ip:port\tproxy to use when connecting to OpenZiti controller. 'http' is currently the only supported type.\n"
-        "\t-i|--identity\toutput identity file\n"
-        "\t-K|--use-keychain\tuse keychain to generate/store private key\n"
-        "\t-k|--key\tprivate key for enrollment\n"
-        "\t-c|--cert\tcertificate for enrollment\n"
-        "\t-n|--name\tidentity name\n",
-        parse_enroll_opts, enroll);
+static CommandLine enroll_cmd = make_command(
+    "enroll", "enroll Ziti identity",
+    "( -u|--url <controller URL> | -j|--jwt <enrollment token> ) -i|--identity <identity> [-k|--key <private_key> [-c|--cert <certificate>]] [-n|--name <name>]",
```
Comment (dovholuknf) [inline programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2370]: needs a `\n` on it doesn't it?

**Review by dovholuknf — APPROVED** [review]
pls check the one `\n`, lgtm othewise

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c (hunk 2 above)
Comment (ekoby) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2370]: no

**Pull request merged**